mercury/static/mercury/style.css

```diff
@@ -263,29 +268,10 @@ input#id_can_msg{
 }
 
 .tooltip {
```

Comment:
It isn't really a tooltip anymore. I would rename this class if you insist on removing tooltips.

Reply:
Will rename. I wasn't able to find a way to make the tooltips display when hovered over that didn't clip outside the window when it was less than full size so I opted to remove them and try to make the text more concise.

Reply:
I'm not sure what you mean by clip outside the window. Can you explain more? The tooltips for me were all within the window.

Reply:
<img width="1680" alt="Screen Shot 2019-12-11 at 3 22 05 PM" src="https://user-images.githubusercontent.com/22565607/70657612-2305ac80-1c2a-11ea-8f6d-ca52ba890bf6.png">

If the window is not fullscreen the tooltip text gets cut off by the edge of the window.

Reply:
I think we should keep the tooltips as Dan mentioned in his feedback that there was too much text on the index page. "There's a lot of text on the landing page which is cumbersome to read."

Reply:
You can just put a `<br>` before the `<span>` if you want to put the tooltip underneath the "button" rectangle.

Reply:
![image](https://user-images.githubusercontent.com/16791467/70657852-af17d400-1c2a-11ea-8881-6efa11bb7976.png)

Reply:
I don't think tooltips that appear when you hover over the text are a good fix to Dan's feedback. It gives the text boxes messy, dual functionalities. The boxes are now buttons that can be clicked but they also generate messages when hovered over. The tooltip bubble doesn't fit the style of the rest of the site, it obscures the names/descriptions of other buttons when active, and when tooltips are underneath the text, and they block clicking for other buttons.

In your picture, if you activate the stopwatch tooltip like in the above picture and move your cursor over the "R" in "Raw Telemetry" you'll see the issue. Even though you're over Raw Telemetry, its tooltip doesn't activate, Stopwatch's stays open and if you click, even though the cursor is over Raw Telemetry, you'll be directed to the Stopwatch page.

I think 2 solutions are possible, and both involve removing tooltips:
- Put all text that would be in tooltips into its description and make the text as succinct as possible. Doing this only adds a few lines total to the homepage. From a UX perspective, the user must now only scroll to get all the info, rather than move the cursor and hover over each item.
- Keep only the teaser text on the home page and make sure the descriptions on each page are sufficient, which are encapsulated within accordion text boxes.

Reply:
I’m not pro-tooltips, but I’m pro-less_text, so I’d vote for the second. 

Style can be updated, so I don’t think that’s a really strong argument, and I don’t think it’s really messy. It shows you text when you hover over it and hyperlinks are followed underneath your cursor. I do think all of your concerns could be addressed by better (proper) implementation and still use tool tips (since mine was fast and dirty due to time), but this isn’t work I have time to do on the last day. I think we’re all better off not adding more text to the landing page that is immediately visible.

Reply:
Ok, I'll reduce the text to single-line descriptions and make sure the page descriptions are sufficient.

Reply:
I agree tool tips can be useful and could have been cleaned up given more time. Possibly limiting them to activate only over an info icon (https://material.io/resources/icons/?icon=info&style=baseline) next to each page title